.github/workflows/CICD.yml

```
@@ -625,7 +625,7 @@ jobs:
       uses: actions-rs/install@v0.1
       with:
         crate: grcov
-        version: latest
+        version: 0.8.0
```

```suggestion
        version: latest
```

grcov 0.8.2 fixes the issue I tihnk